src/org/zaproxy/zap/extension/spider/SpiderPanel.java

```diff
@@ -310,40 +383,50 @@ private JLabel getFoundCountValueLabel() {
 		return foundCountValueLabel;
 	}
 
+
+    /**
+     * Gets the label storing the name of the count of added URIs.
+     * 
+     * @return the found count name label
+     */
+    private JLabel getAddedCountNameLabel() {
+        if (addedCountNameLabel == null) {
+            addedCountNameLabel = new javax.swing.JLabel();
+            addedCountNameLabel.setText(Constant.messages.getString("spider.toolbar.added.label"));
+        }
+        return addedCountNameLabel;
+    }
+
+    /**
+     * Gets the label storing the value for count of added URIs.
+     * 
+     * @return the added count value label
+     */
+    private JLabel getAddedCountValueLabel() {
+        if (addedCountValueLabel == null) {
+            addedCountValueLabel = new javax.swing.JLabel();
+            addedCountValueLabel.setText(ZERO_REQUESTS_LABEL_TEXT);
+        }
+        return addedCountValueLabel;
+    }
+
 	@Override
 	protected int addToolBarElements(JToolBar toolBar, Location location, int gridX) {
 		if (ScanPanel2.Location.afterProgressBar == location) {
 			toolBar.add(new JToolBar.Separator(), getGBC(gridX++, 0));
 			toolBar.add(getFoundCountNameLabel(), getGBC(gridX++, 0));
 			toolBar.add(getFoundCountValueLabel(), getGBC(gridX++, 0));
+
 			toolBar.add(new JToolBar.Separator(), getGBC(gridX++, 0));
-			toolBar.add(getShowMessagesToggleButton(), getGBC(gridX++, 0));
+            toolBar.add(getAddedCountNameLabel(), getGBC(gridX++, 0));

```

Comment:
I'm going to trust that these stick out like a sore thumb in the GitHub diff, and that you're going to catch them. If you want me to comment them all then say so and I'll nitpick it later :grin: